spec/helpers/sellers_helper_spec.rb

```diff
@@ -88,7 +100,7 @@
   describe '#calculate_amount_raised' do
     context 'with no money raised' do
       it 'returns the list of sellers with `and`' do
```

Comment:
You should also write a test to make sure expected gift card count and donation count is correct